### ISSUE TITLE
Mixed slash and block comment parsing creates a syntax error.

### ISSUE THREAD

**Issue by JonKrone** [postcss/postcss-scss#48]
Hi! While linting some legacy code I noticed a missed case for converting `//` comments into `/* */` style comments. When a line has mixed slash and block comments (e.g. `// top: 0 /* top: 50px */`), wrapping the whole line in a block comment produces a syntax error because the outer block comment is ended by the first `*/`, leaving later `*/` pieces without a matching opening piece.

Minimal replication:
```
.class {
    // width: 50%; /* width: 66%; */
}
```

expected:
leave as is, or
```
.class {
    /* width: 50%; */ /* width: 66%; */
}
```

actual:
```
.class {
    /* width: 95.2%; /*height: 96.3%;*/*/
}
```

By wrapping the whole line in the block comment, we're actually left with two block-ending `*/` segments, which is a syntax error.

I took me some time to track down the cause of the automatic formatting because I also have other plugins that deal with comments and formatting and I did not expect the scss parser to format as well. As a workaround, I am trying to disable this behavior but in the meantime I wanted to bring this up with y'all.

I am not the most familiar with css vs scss syntax, am I missing something or is this a bug?

**Comment by ai** [comment]
You could use https://github.com/mummybot/postcss-strip-inline-comments as fast fix.

I will try to find some solution soon.

**Comment by ai** [comment]
Done 8713aa8

**Comment by ai** [comment]
Released in 0.4.

**Comment by JonKrone** [comment]
ai! Awesome!! I spent a couple hours yesterday trying to resolve it and ended up mucking about in scss-parse filtering out inline comments but your fix will work great.

Thanks for getting right on it :)
